guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore.adoc

```diff
@@ -0,0 +1,43 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore"]
+= Preparing for disaster recovery with active and passive {ProjectServer} and backup and restore
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
```

Comment:
@evgeni Is the cloning necessary? Or would you think of it like an efficiency thing?

Reply:
I checked my notes and it's true that cloning is not mentioned there. This might be a semi-accidental leftover from the external storage scenario.

So if cloning is not necessary, what do I replace it with? A simple restore on a clean system?